plugins/modules/mongodb_user.py

```diff
@@ -215,10 +215,13 @@ def user_add(module, client, db_name, user, password, roles):
         # Should allow us to create the first user. So we assume this is the case
         # Might be able to do something with db.getUsers() if this approach needs
         # a rethink
-        if excep.code == 13:  # Unauthorized
-            exists = False
+        if hasattr(excep, 'code'):
+            if excep.code == 13:  # Unauthorized
+                exists = False
+            else:
+                raise excep
         else:
-            raise Exception
+            raise excep
```

Comment:
```suggestion
            raise
```

Reply:
Bad habit from my Java days. I'll probably be doing this everywhere.